generated/nirfmxbt/nirfmxbt.proto

```diff
@@ -31,27 +325,1873 @@ enum Boolean {
+  NIRFMXBT_INT32_DIGITAL_EDGE_TRIGGER_EDGE_RISING_EDGE = 0;
+  NIRFMXBT_INT32_DIGITAL_EDGE_TRIGGER_EDGE_FALLING_EDGE = 1;
+  NIRFMXBT_INT32_DIRECTION_FINDING_MODE_DISABLED = 0;
+  NIRFMXBT_INT32_DIRECTION_FINDING_MODE_ANGLE_OF_ARRIVAL = 1;
+  NIRFMXBT_INT32_DIRECTION_FINDING_MODE_ANGLE_OF_DEPARTURE = 2;
+  NIRFMXBT_INT32_FREQUENCY_RANGE_AVERAGING_ENABLED_FALSE = 0;
+  NIRFMXBT_INT32_FREQUENCY_RANGE_AVERAGING_ENABLED_TRUE = 1;
+  NIRFMXBT_INT32_IQ_POWER_EDGE_TRIGGER_LEVEL_TYPE_RELATIVE = 0;
+  NIRFMXBT_INT32_IQ_POWER_EDGE_TRIGGER_LEVEL_TYPE_ABSOLUTE = 1;
+  NIRFMXBT_INT32_IQ_POWER_EDGE_TRIGGER_SLOPE_RISING_SLOPE = 0;
+  NIRFMXBT_INT32_IQ_POWER_EDGE_TRIGGER_SLOPE_FALLING_SLOPE = 1;
+  NIRFMXBT_INT32_LIMITED_CONFIGURATION_CHANGE_DISABLED = 0;
+  NIRFMXBT_INT32_LIMITED_CONFIGURATION_CHANGE_NO_CHANGE = 1;
+  NIRFMXBT_INT32_LIMITED_CONFIGURATION_CHANGE_FREQUENCY = 2;
+  NIRFMXBT_INT32_LIMITED_CONFIGURATION_CHANGE_REFERENCE_LEVEL = 3;
+  NIRFMXBT_INT32_LIMITED_CONFIGURATION_CHANGE_FREQUENCY_AND_REFERENCE_LEVEL = 4;
+  NIRFMXBT_INT32_LIMITED_CONFIGURATION_CHANGE_SELECTED_PORTS_FREQUENCY_AND_REFERENCE_LEVEL = 5;
+  NIRFMXBT_INT32_MOD_ACC_AVERAGING_ENABLED_FALSE = 0;
+  NIRFMXBT_INT32_MOD_ACC_AVERAGING_ENABLED_TRUE = 1;
+  NIRFMXBT_INT32_MOD_ACC_BURST_SYNCHRONIZATION_TYPE_NONE = 0;
+  NIRFMXBT_INT32_MOD_ACC_BURST_SYNCHRONIZATION_TYPE_PREAMBLE = 1;
+  NIRFMXBT_INT32_MOD_ACC_BURST_SYNCHRONIZATION_TYPE_SYNC_WORD = 2;
+  NIRFMXBT_INT32_MOD_ACC_IQ_ORIGIN_OFFSET_CORRECTION_ENABLED_FALSE = 0;
+  NIRFMXBT_INT32_MOD_ACC_IQ_ORIGIN_OFFSET_CORRECTION_ENABLED_TRUE = 1;
+  NIRFMXBT_INT32_PACKET_TYPE_UNKNOWN = -1;
+  NIRFMXBT_INT32_PACKET_TYPE_DH1 = 0;
+  NIRFMXBT_INT32_PACKET_TYPE_DH3 = 1;
+  NIRFMXBT_INT32_PACKET_TYPE_DH5 = 2;
+  NIRFMXBT_INT32_PACKET_TYPE_DM1 = 3;
+  NIRFMXBT_INT32_PACKET_TYPE_DM3 = 4;
+  NIRFMXBT_INT32_PACKET_TYPE_DM5 = 5;
+  NIRFMXBT_INT32_PACKET_TYPE_2_DH1 = 6;
+  NIRFMXBT_INT32_PACKET_TYPE_2_DH3 = 7;
+  NIRFMXBT_INT32_PACKET_TYPE_2_DH5 = 8;
+  NIRFMXBT_INT32_PACKET_TYPE_3_DH1 = 9;
+  NIRFMXBT_INT32_PACKET_TYPE_3_DH3 = 10;
+  NIRFMXBT_INT32_PACKET_TYPE_3_DH5 = 11;
+  NIRFMXBT_INT32_PACKET_TYPE_2_EV3 = 12;
+  NIRFMXBT_INT32_PACKET_TYPE_2_EV5 = 13;
+  NIRFMXBT_INT32_PACKET_TYPE_3_EV3 = 14;
+  NIRFMXBT_INT32_PACKET_TYPE_3_EV5 = 15;
+  NIRFMXBT_INT32_PACKET_TYPE_LE = 16;
+  NIRFMXBT_INT32_PAYLOAD_BIT_PATTERN_STANDARD_DEFINED = 0;
+  NIRFMXBT_INT32_PAYLOAD_BIT_PATTERN_11110000 = 1;
+  NIRFMXBT_INT32_PAYLOAD_BIT_PATTERN_10101010 = 2;
+  NIRFMXBT_INT32_PAYLOAD_LENGTH_MODE_MANUAL = 0;
+  NIRFMXBT_INT32_PAYLOAD_LENGTH_MODE_AUTO = 1;
+  NIRFMXBT_INT32_TRIGGER_MINIMUM_QUIET_TIME_MODE_MANUAL = 0;
+  NIRFMXBT_INT32_TRIGGER_MINIMUM_QUIET_TIME_MODE_AUTO = 1;
+  NIRFMXBT_INT32_TRIGGER_TYPE_NONE = 0;
+  NIRFMXBT_INT32_TRIGGER_TYPE_DIGITAL_EDGE = 1;
+  NIRFMXBT_INT32_TRIGGER_TYPE_IQ_POWER_EDGE = 2;
+  NIRFMXBT_INT32_TRIGGER_TYPE_SOFTWARE = 3;
+  NIRFMXBT_INT32_TWENTYD_B_BANDWIDTH_AVERAGING_ENABLED_FALSE = 0;
```

Comment:
Something to add to a future task (the API review story?) These names are messed up: TWENTYD_B.

Reply:
^^ Same goes for all of the proto comments. They just need a follow up task somewhere. Shouldn't block this PR or User Story.

Reply:
Created [AB#1812664](https://ni.visualstudio.com/DevCentral/_workitems/edit/1812664).